src/ec_talk.erl

```diff
@@ -37,7 +37,9 @@
          ask_default/3,
          ask/3,
          say/1,
-         say/2]).
+         say/2,
+	 get_boolean/1,
+	 get_integer/1]).
```

Comment:
same comment as last time this happened; if this is an optional export for tests, it should be wrapped in an `-ifdef(TEST).`

Reply:
Yes, as in d24ad720347b72b745c9e3492e61e3f4d0829501. Changed it.